spaces/gmt/skin.scss

```
@@ -0,0 +1,9 @@
+.egg {
```

The css selector should similar to the space key.
```suggestion
.gmt {
```